samples/ImplicitFlow/AuthorizationServer/Controllers/AccountController.cs

```diff
@@ -0,0 +1,312 @@
+﻿using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Mvc.Server.Models;
+using Mvc.Server.Services;
+using Mvc.Server.ViewModels.Account;
+
+namespace Mvc.Server.Controllers {
```

Comment:
Consider renaming the namespace to match the project name. It would be confusing otherwise.